go-controller/pkg/ovn/obj_retry_master.go

```diff
@@ -787,14 +787,14 @@ func (h *masterEventHandler) SyncFunc(objs []interface{}) error {
 		case factory.EgressFirewallType:
 			syncFunc = h.oc.syncEgressFirewall
 
-		case factory.EgressIPType:
+		case factory.EgressIPNamespaceType:
```

Comment:
I think it wouldn't because we call WatchNamespacesForEgressIPs() which is fetching everything from the informers and then calculating everything from scratch, so logic is as is in the  generateCacheForEgressIP